server/routes/locations.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require('supertest')
+const { appWithAllRoutes } = require('./testutils/appSetup')
+
+const locationService = {
+  getActiveAgenciesByType: jest.fn(),
+}
+
+let app
+
+beforeEach(() => {
+  app = appWithAllRoutes({ locationService })
+  locationService.getActiveAgenciesByType.mockResolvedValue([])
+})
+
+afterEach(() => {
+  jest.resetAllMocks()
+})
+
+describe('GET /change-prison', () => {
+  test('should display page title', () => {
+    return request(app)
+      .get(`/report/-19/change-prison`)
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        expect(res.text).toContain('What prison did the use of force take place in?')
```

Comment:
might want to check that a prisoned returned by change prison is present as a select option

Reply:
done
new test added